### PR TITLE
style(mobility): add header logos, buttons

### PR DESCRIPTION
**Pull request by chaseMo**
Screen Shot 2020-07-18 at 12 24 57 PM

### CODE REVIEW

File: src/google-mobility.html [hunk 1]
```diff
@@ -11,23 +11,34 @@
...
               <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>What is RDS?
             </a>
+
+            <a class="rds-button-flat-white" href="https://www.google.com/covid19/mobility/" target="_blank">
+              <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>About Google Community Reports
+            </a>
+            <p class="banner-footnote mdc-typography--caption">
+              This website is intended as a technology showcase and should not be used for any other purposes.
+            </p>
+            <a href="#" class="app-logo"><img class="brand-logo" src="./assets/RDS-Logo.png" /></a>
```
Comment (wallace41290) [inline src/google-mobility.html:31]: 1. This should link to https://www.richdataservices.com/
2. Can you use the green and white logo like on the index.html

File: src/google-mobility.html [hunk 2]
```diff
@@ -11,23 +11,34 @@
...
+              <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>RDS COVID-19 Project</a
+            >
             <a class="rds-button-flat-white" href="https://www.richdataservices.com" target="_blank">
               <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>What is RDS?
             </a>
+
+            <a class="rds-button-flat-white" href="https://www.google.com/covid19/mobility/" target="_blank">
+              <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>About Google Community Reports
+            </a>
+            <p class="banner-footnote mdc-typography--caption">
```
Comment (wallace41290) [inline src/google-mobility.html:28]: On large screen, the text displays next to the button. 
Screen Shot 2020-07-18 at 1 14 17 PM

File: src/google-mobility.html [hunk 3]
```diff
@@ -11,23 +11,34 @@
       <div class="rds-hero" id="home">
         <div class="container mdc-layout-grid">
           <div class="header">
+            <a href="https://www.google.com/covid19/mobility/"><img class="google-logo" src="./assets/Google__G__Logo.svg" /></a>
```
Comment (wallace41290) [inline src/google-mobility.html:14]: I'm going to send you an updated svg that has a white background to try.

**Comment by chaseMo** [comment]
Screen Shot 2020-07-18 at 7 15 43 PM

**Pull request merged**